cli/cmd/volume.go

```diff
@@ -78,7 +79,18 @@ var volumeCreateCmd = &cobra.Command{
 		validateNArgs(cmd, 2, 0)
 		volname := cmd.Flags().Args()[0]
 		bricks := cmd.Flags().Args()[1:]
```

Comment:
Can you please check if this will work when `replica` and `transport` keywords are specified in the CLI.

Reply:
replica and transport are flags. --replica, --transport etc. So it will work.

Callback func has two arguments, `func(cmd *cobra.Command, args []string)` second argument contains os.Args, but cmd.Flags().Args() contains only arguments excluding all the flags. So this approach should work.

https://godoc.org/github.com/spf13/pflag#FlagSet